packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -67,7 +67,7 @@ export const ACTIVE = 'ACTIVE';
 
 export const BUSY = 'BUSY';
 
-export const CALL = 'CALL';
+export const _CALL_ = 'CALL';
```

Comment:
Is there something more descriptive we can call this? Or group it as a property within another object, like `LOCUS.CALL`?